### PR TITLE
 Updates to indel calling and reporting script and added a blacklisted variant

### PR DESCRIPTION
**Pull request by dhspence**

### CODE REVIEW

File: Chromoseq.wdl [hunk 1]
```diff
@@ -232,9 +265,10 @@ workflow ChromoSeq {
     annotate_svs.vcf_index,
     annotate_svs.allvcf,
     annotate_svs.allvcf_index,
-    run_varscan.varscan_snv_file,
-    run_varscan.varscan_indel_file,
-    run_detect_flt3itd.vcf,
```
Comment (dufeiyu) [inline Chromoseq.wdl:237]: Do we need gather outputs of run_varscan_snv.vcf, run_varscan_indel.vcf, run_pindel_indels.vcf, run_manta_indels.vcf ?

Comment (dhspence) [reply Chromoseq.wdl:237]: All variants with any support should be included in the final annotated_filtered VCF so I dont think we need to gather these other outputs.

File: Chromoseq.wdl [hunk 2]
```diff
@@ -434,9 +465,40 @@ task run_varscan {
   command <<<
     /usr/local/bin/samtools mpileup -f ${refFasta} -l ${CoverageBed} ${Bam} > ${tmp}/mpileup.out && \
     java -Xmx12g -jar /opt/varscan/VarScan.jar mpileup2snp ${tmp}/mpileup.out --min-coverage ${default=6 MinCov} --min-reads2 ${default=3 MinReads} \
-    --min-var-freq ${default="0.02" MinFreq} --p-value ${default="0.01" pvalsnv} --output-vcf | /opt/conda/bin/bgzip -c > ${Name}.snv.vcf.gz && /opt/conda/bin/tabix ${Name}.snv.vcf.gz && \
+    --min-var-freq ${default="0.02" MinFreq} --p-value ${default="0.01" pvalsnv} --output-vcf | /opt/conda/bin/bgzip -c > ${Name}.varscan_snv.vcf.gz && /opt/conda/bin/tabix ${Name}.varscan_snv.vcf.gz
```
Comment (dufeiyu) [inline Chromoseq.wdl:468]: Is "tabix -p vcf" needed here ?

File: Chromoseq.wdl [hunk 3]
```diff
@@ -447,39 +509,63 @@ task run_varscan {
...
+  
+  runtime {
+    docker_image: docker
+    cpu: "1"
+    memory: "16 G"
+    queue: queue
+    job_group: jobGroup
+  }
+  output {
+    File vcf = "${Name}.pindel.vcf.gz"
```
Comment (dufeiyu) [inline Chromoseq.wdl:545]: Is vcf_index needed here ?

File: Chromoseq.wdl (hunk 2 above)
Comment (dhspence) [reply Chromoseq.wdl:468]: Yes, good idea. Not absolutely needed, but add it for good measure.

File: Chromoseq.wdl (hunk 3 above)
Comment (dhspence) [reply Chromoseq.wdl:545]: Yes, thats fine.

File: inputs.local.json [hunk 4]
```diff
@@ -9,12 +9,13 @@
...
+    "ChromoSeq.GenesBed": "/gscmnt/gc2555/spencer/dhs/git/cle-chromoseq/accessory_files/chromoseq_genes.bed",
     "ChromoSeq.Cytobands": "/gscmnt/gc13016/cle/54f8f7b915cb472aa183c721307369ab_scratch_space/ChromoSeq/git/cle-chromoseq/accessory_files/hg38.cytoBandIdeo.bed.gz",
     "ChromoSeq.SVDB": "/gscmnt/gc13016/cle/54f8f7b915cb472aa183c721307369ab_scratch_space/ChromoSeq/git/cle-chromoseq/accessory_files/chromoseq_sv_filter.bedpe.gz",
-    "ChromoSeq.CustomAnnotationVcf":"/gscmnt/gc13016/cle/54f8f7b915cb472aa183c721307369ab_scratch_space/ChromoSeq/git/cle-chromoseq/accessory_files/chromoseq_custom_anntations.040920.vcf.gz",
-    "ChromoSeq.CustomAnnotationIndex":"/gscmnt/gc13016/cle/54f8f7b915cb472aa183c721307369ab_scratch_space/ChromoSeq/git/cle-chromoseq/accessory_files/chromoseq_custom_anntations.040920.vcf.gz.tbi",
-    "ChromoSeq.CustomAnnotationParameters":"MYELOSEQ,vcf,exact,0,TCGA_AC,MDS_AC,MYELOSEQBLACKLIST",
+    "ChromoSeq.CustomAnnotationVcf":"/gscmnt/gc2555/spencer/dhs/git/cle-chromoseq/accessory_files/chromoseq_custom_annotations.111720.vcf.gz",
```
Comment (dufeiyu) [inline inputs.local.json:15]: I would suggest to rename "chromoseq_custom_annotations.111720.vcf.gz" to "chromoseq_custom_annotations.vcf.gz" and make the change to inputs.local.json and Dockerfile accordingly.

Comment (dhspence) [reply inputs.local.json:15]: Sure, thats fine. Good idea.

Comment (dhspence) [reply inputs.local.json:15]: Also, there is one new edit that I dont see here. Do I need to make another pull request or update this one?

Comment (dufeiyu) [reply inputs.local.json:15]: git commit your new edit in your local git (where this PR comes from), then git push (to remote fork). The update should be included in this PR.  No need for new PR

**Comment by dufeiyu** [comment]
Hi Dave, this PR looks good to me if the test workflow of Chromoseq.wdl succeeds. There are several places that might need "tabix -p vcf" and output vcf .tbi index. It would be good to rename with "chromoseq_custom_annotations.vcf.gz".  There are also several places with "gscmnt" of cle scratch_space and gc2555/Spencer/dhs. If you really need those for testing, you can leave them as is and merge the PR. I can make new PRs to fix those paths and add my own changes.

**Comment by dhspence** [comment]
Thanks. I think we should definitely change the paths of the scripts to the docker path. Renaming the custom annotation VCF is also fine and adding '-p vcf' is also fine. I dont think we need to pass the VCF indexes to each task, but we can at least make it an output.

**Pull request merged**